src/test/java/com/google/step/snippet/external/W3SchoolsClientTest.java

```diff
@@ -15,19 +15,20 @@
 
   @Test
   public void htmlCodeCard() {
-    Card actual = client.search("https://www.w3schools.com/tags/tag_img.asp");
+    Card actual = client.search("https://www.w3schools.com/tags/tag_img.asp", "html img");
     Card expected =
         new Card(
-            "HTML <img> Tag",
-            "<img src=\"img_girl.jpg\" alt=\"Girl in a jacket\" width=\"500\" height=\"600\">",
+            "HTML \n&lt;img&gt; Tag",
```

Comment:
So the div tag removal thing seems resolved, but Jsoup.clean is still stripping away class settings, and style settings in a tag. 

In the htmlCodeCard test, it's stripping away the `src=` field, in cssEscaping `style=` is being stripped, and in divEscaping `class=`.

I wrote up some tests that should help check this. 
```
 @Test
  public void htmlCodeCard() {
    Card actual = client.search("https://www.w3schools.com/tags/tag_img.asp", "html img");
    Card expected =
        new Card(
            "HTML \n&lt;img&gt; Tag",
            "&lt;img src=&quot;img_girl.jpg&quot; alt=&quot;Girl in a jacket&quot; width=&quot;500&quot; height=&quot;600&quot;&gt;",
            "https://www.w3schools.com/tags/tag_img.asp",
            "How to insert an image:");
    assertEquals(expected, actual);
  }
```
```
  @Test
  public void cssEscaping() {
    Card actual = client.search("https://www.w3schools.com/html/html_css.asp", "css style");
    Card expected =
        new Card(
            "HTML Styles - CSS",
            "&lt;h1 style=&quot;color:blue;&quot;&gt;A Blue Heading&lt;/h1&gt; &lt;p style=&quot;color:red;&quot;&gt;A red paragraph.&lt;/p&gt;",
            "https://www.w3schools.com/html/html_css.asp",
            "CSS stands for Cascading Style Sheets.");
    assertEquals(expected, actual);
  }
```
```
  @Test
  public void divEscaping() {
    Card actual = client.search("https://www.w3schools.com/css/css3_flexbox.asp", "css flexbox");
    Card expected =
        new Card(
            "CSS Flexbox",
            "&lt;div class=&quot;flex-container&quot;&gt; &lt;div&gt;1&lt;/div&gt; &lt;div&gt;2&lt;/div&gt; &lt;div&gt;3&lt;/div&gt; &lt;/div&gt;",
            "https://www.w3schools.com/css/css3_flexbox.asp",
            "Try it Yourself &raquo;");
    assertEquals(expected, actual);
  }
}
```

Reply:
I was aware of the stripping being done to these fields, as (I believe) clean/Whitelist has its own heuristics for what constitutes a "safe" param for these fields. Example: for the src field, only http and https links are allowed. However, I know that our definition of "safe" might be different for the sake of preserving fields like this. Thoughts?